week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,39 @@
-function setAlarm() {}
+function setAlarm() {
+  let count = document.getElementById("timeRemaining");
+  let setIt = document.getElementById("set");
+  let pause = document.getElementById("p");
+  let Restart = document.getElementById("re");
+  let Counter;
+  pause.addEventListener("click", function () {
+    clearInterval(Counter);
+  });
+  Restart.addEventListener("click", function () {
+    Counter = setInterval(CountDown, 1000);
+  });
+  let TheTime;
+  setIt.addEventListener("click", function () {
+    TheTime = document.getElementById("alarmSet").value;
+
+    if (TheTime <= 60) {
+      count.innerHTML = "Time Remaining: 00:" + TheTime;
```

Comment:
Time should be padded to 2-character long. So when time is smaller than 10, a `0` should be prepended e.g. `06`.
You can use the `padStart` built-in JS function to do that.